model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/PushMessageInformation.java

```diff
@@ -36,6 +36,10 @@
     private Date submitDate = new Date();
     private long totalReceivers;
 
+    private long appOpenCounter;
+    private Date firstOpenDate;
```

Comment:
Wouldn't it be better to get this information from a query instead of duplicating it, it will not be true once updated
![metrics](https://cloud.githubusercontent.com/assets/51133/7136140/dfe1ad60-e2ad-11e4-823a-b44815a54256.png)